src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -283,7 +285,8 @@ trait EntityPage extends HtmlPage {
     val template: Elems = List(
      Div(id="template", elems= List(
         Div(id="allMembers", elems=
-          memsDiv("members", "Instance Constructors", constructors, "constructors")
+             memsDiv("package members", "Package Members", packageMembers, "packages")
```

Comment:
I agree we should add a list of package members, but I think you'd expect to first see type and value members, before listing packages. In 2.11, packages were listed as part of value members (which is also not right, since they are not values, strictly speaking, but the ordering makes more sense).

Reply:
Actually, I think I would quite like to see subpackages listed first. If I'm diving down to find a given class in a sub-sub-package, I would want to always find subpackages first so that I do not have to scroll down (too much) at every level. Instead, I would only scroll once at the deepest level to find the class I'm interested in.

This is similar to how most file explorers will list subfolders first, irrespective of the chosen ordering.

Reply:
True